src/amo/components/SearchResult.js

```diff
@@ -28,11 +28,12 @@ export class SearchResultBase extends React.Component {
             <div className="SearchResult-rating">
               <Rating rating={addon.ratings.average} readOnly size="small" />
             </div>
-            <h3 className="SearchResult-author">{addon.authors[0].name}</h3>
-            <h3 className="SearchResult-users">{i18n.sprintf(
-              i18n.ngettext('%(total)s user', '%(total)s users', averageDailyUsers),
-              { total: i18n.formatNumber(averageDailyUsers) },
-            )}
+            <h3 className="SearchResult-author-users">
+              <span className="SearchResult-author">{addon.authors[0].name}</span>
```

Comment:
This code will create issues if the author does not exist. The code to deal with no authors here will be complicated, because the `—` needs to be localised but omitted if there is no author. So it needs to be:

```js
i18n.sprintf(i18n.ngettext(
  '%(addonAuthor)s%(separator)s%(addonUsers)s user', 
  '%(addonAuthor)s%(separator)s%(addonUsers)s users', 
  averageDailyUsers,
), {
  addonAuthor: addon.authors.length ? addon.authors[0].name : null,
  separator: addon.authors.length ? i18n.gettext(' — ') : null,
  addonUsers: i18n.formatNumber(averageDailyUsers),
});
```